source/_integrations/roomba.markdown

```diff
@@ -88,3 +88,9 @@ Use this credentials in dorita980 lib :)
 ```
 
 Copy the password (everything between `=>` and `<=`, not including leading and trailing whitespace) into the Home Assistant password dialog.
+
+## Troubleshooting
+
+- **Integration wizard shows "Failed to connect" after submitting the password**: Before attempting a factory reset (which can be a cumbersone process), attempt submitting the password in the integration wizard while the Roomba is actively running (i.e. cleaning). Avoid opening the app to start a manual job to help with this. Instead, push the physical clean button on the device directly to start the manual job. This appears to resolve the issue on some models because they answer queries only while actively running ([Github thread](https://github.com/home-assistant/core/issues/117071#issuecomment-2295765878)).
```

Comment:
Let's remove the reference to GitHub issues. It shouldn't be needed. The documentation described all there is to it.

../Frenck

Reply:
That makes sense. I'll push an update.